content/en/BestPractices/insecure-deserialization.md

```diff
@@ -0,0 +1,290 @@
+---
+Title: "Insecure Deserialization"
+linkTitle: "Insecure Deserialization"
+weight: 660
+description: >
+  Insecure Deserialization is one of the OWASP Top 10 web vulnerabilities for 2017. OWASP has included this issue in the [OWASP Top 10:2021](https://owasp.org/Top10/) vulnerabilities as [Software and Data Integrity Failures](https://owasp.org/Top10/A08_2021-Software_and_Data_Integrity_Failures/).
```

Comment:
```suggestion
  Insecure Deserialization is one of the OWASP Top 10 web vulnerabilities for 2017. OWASP has included this issue in the [OWASP Top 10:2021](https://owasp.org/Top10/) vulnerabilities as [Software and Data Integrity Failures](https://owasp.org/Top10/A08_2021-Software_and_Data_Integrity_Failures/).
```

This description is too long. We usually use a short sentence.

Reply:
It's two separate sentences.

Reply:
@mjang-cobalt Yes, but it looks terrible in previews and doesn't align with other articles.

![Screenshot 2022-12-06 at 20 45 10](https://user-images.githubusercontent.com/106706890/206007843-57e17098-dc8f-467c-a69c-ec6a1d817298.png)